js/UserPic.js

```diff
@@ -89,7 +93,7 @@ module.exports = React.createClass({
     },
 
     getUserInfo(callback) {
-        forceClient.sendRequest('/services/data', '/v36.0/chatter/users/me', 
+        net.sendRequest('/services/data', '/v36.0/chatter/users/me', 
```

Comment:
Should we switch this to `v41.0` to be inline with `6.0`?

Reply:
Will do.